sympy/physics/mechanics/kane.py

```diff
@@ -534,7 +534,7 @@ def get_partial_velocity(body):
         self._f_d = -(self._fr - nonMM)
         return fr_star
 
-    def to_linearizer(self, linear_solver='LU'):
+    def to_linearizer(self, linear_solver='LU', jacobian_func='forward_jacobian'):
```

Comment:
In the meeting I suggested `jacobian_func=None` as a default to avoid having to name the default. In the long term, I think we should only have a single optimized jacobian in sympy, this is just a temp performance boost with this new one and we may always keep it private.

Reply:
And default should be the normal "classic" sympy jacobian.